reference_guide/ui_themes/themes_customize.md

```diff
@@ -166,6 +178,49 @@ The following example sets the background color for all labels to the color #F6E
 
+#### Customizing the Color of UI Tabs
+The color of UI Tabs are changed by adding a key-value pair to the `"ui": {}` section of a Theme description file.
+The `key` is the `element.property` format and the `value` is the custom color.
+
+There are two implementations of tabs in the IntelliJ Platform:
+* Editor Tabs, which e.g., represent open files in the [Editor window](https://www.jetbrains.com/help/idea/using-code-editor.html), and in [Tool Window bars](https://www.jetbrains.com/help/idea/tool-windows.html#bars_and_buttons).
+* Tabbed Panes, which e.g., are used for the [Run/Debug Configurations dialog](https://www.jetbrains.com/help/idea/run-debug-configurations-dialog.html).
+
+The control keys for UI Tabs were expanded from build 191 to 192 of the IntelliJ Platform.
+The build 191 control keys are compatible with build 192 and later versions of the IntelliJ Platform. 
+
+| Build 191 Element | Build 192 Element | Description of 192 Element |
+|------|---------|---------|
+| N/A  | **`DefaultTabs`** | Applied to all tabs except `TabbedPane`, _unless_ overridden by a more specific Tab control element. This element is the background under all tabs. |
+| **`EditorTabs`**  | **`EditorTabs`** | Applied only to Editor tabs. Overrides any `DefaultTab` settings. 192 has many more `property` settings than 191. |
+| **`ToolWindow.HeaderTab`** | **`ToolWindow`** | Applied only to Tool Window tabs. Overrides any `DefaultTab` settings. 192 has many more `property` settings than 191. |
+| **`DebuggerTabs`** | **`DefaultTabs`** | `DefaultTab` settings are used instead of `DebuggerTabs`, except for key `DebuggerTabs.underlineHeight`. |
+| N/A | **`TabbedPane`** | New in 192. Applied only to Tabbed Panes. |
+| **`Plugins.Tab`** | **`TabbedPane`** | Use `TabbedPane` instead. |
+| **`SearchEverywhere.Tab`** | **`SearchEverywhere.Tab`** | No change. |
+
+Methods for identifying UI control keys are in the [Finding Attribute Keys for UI Controls](#finding-attribute-keys-for-ui-controls) section. 
+
+For example, some Editor Tab colors are set in this snippet for build 192 and later.
+Note the changes from build 191 properties, which are updated from an excerpt from the IntelliJ Platform [High Contrast Theme](upsource:///platform/platform-resources/src/themes/HighContrast.theme.json?nav=2730:2734:focused&line=199&preview=false):
+This update is done solely for documentation purposes; the keys from build 191 continue to work in build 192:
+* `EditorTabs.underlinedTabForeground` (b192) was `EditorTabs.selectedForeground` (b191)
+* `EditorTabs.underlinedTabBackground` (b192) was `EditorTabs.selectedBackground` (b191)
+* `EditorTabs.inactiveColoredFileBackground` (b192) was `EditorTabs.inactiveMaskColor` (b191)
+
+```json
+{
+  "ui": {
+    "EditorTabs": {
+      "underlinedTabForeground": "#FFFFFF",
+      "underlinedTabBackground": "#3333FF",
+      "underlineColor": "#3333FF",
+      "inactiveColoredFileBackground": "#000000FF"
+    }
+  }
+}
+```
```

Comment:
This snippet shows only the new keys. I suggest it shows both new and old keys, to make it clear that both can be used. See example in high contrast https://github.com/OlyaB/HighContrastTheme/blob/master/src/HighContrast.theme.json

Reply:
This sample is much more interesting. Did I correctly describe how e.g. `EditorTabs.selectedBackground` is interpreted by the platform in comparison to `EditorTabs.underlinedTabBackground` when both appear in the same JSON file?

Reply:
This is correct, thank you!